S4TF_Tutorial_5.ipynb

```diff
@@ -0,0 +1,1181 @@
+        "3. Convert them into floating point tensors\n",
+        "4. Rescale the tensors from values between 0 and 255 to values between 0 and 1, as neural networks prefer to deal with small input values.\n",
+        "\n",
+        "We have done this in the following code."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "aRAT9M538M82",
+        "colab_type": "text"
+      },
+      "source": [
+        "The `tensor` function takes image path as input and outputs a tuple of resized image tensor and it’s corresponding label.\n",
+        "\n",
+        "The `tensors` function takes in a list of paths and the number of tensors to be produced in the output tensor as input. While processing, it gives number of images done as print statements so that we can have an understanding of how much process is done."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "1DC85ego6kmK",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "// tensor returns the resized image tensor and it's corresponding label\n",
+        "\n",
+        "func tensor(path: String) -> (Tensor<Float>, Int32) {\n",
+        "    let img = pil.Image.open(path)\n",
+        "    let image = np.array(img)*(1.0/255)\n",
+        "    let height = Int(image.shape[0])\n",
+        "    let width = Int(image.shape[1])\n",
+        "    let finalImage = np.zeros([height, width, 3], dtype: np.float32)\n",
+        "    for i in 0..<height! {\n",
+        "        for j in 0..<width! {\n",
+        "            finalImage[i, j] = image[i][j]\n",
+        "        }\n",
+        "    }\n",
+        "    var tensor = Tensor<Float>(numpy: finalImage)!\n",
+        "    tensor = Raw.expandDims(tensor, dim: Tensor<Int32>(0))\n",
+        "    tensor = Raw.resizeArea(images:tensor , size:[150, 150])\n",
+        "    \n",
+        "    var label: Int32 = -1\n",
+        "    if path.contains(\"dog.\") {\n",
+        "        label = 0 \n",
+        "        } else {\n",
+        "        label = 1\n",
+        "    }\n",
+        "    \n",
+        "    return (tensor, label)\n",
+        "}\n",
+        "\n",
+        "//tensors returns the entire list converted to tensor array of images and labels from tensor function\n",
+        "func tensors(fromList: PythonObject, valueCount: Int) -> (Tensor<Float>, Tensor<Int32>) {\n",
+        "    let batchFiles = fromList[0..<valueCount]\n",
+        "    var labels = [Int32]()\n",
+        "    var x: Tensor<Float>\n",
+        "    var y: Tensor<Int32>\n",
+        "    var start: Int\n",
+        "\n",
+        "    //load first image\n",
+        "    let path: String = String(batchFiles[0]) ?? \"\"\n",
+        "    let data = tensor(path: path)\n",
+        "    x = data.0 \n",
+        "    labels.append(data.1)\n",
+        "\n",
+        "    //load the rest images \n",
```

Comment:
- Add a whitespace after `//`.
- Start each comment with a capital letter.
- End each comment with a period.
```suggestion
        "    // Load the rest images.\n",
```
Could you please apply this everywhere?

Reply:
Yes sure